### DISCUSSION TITLE
くじ引きのモーダル化

### DISCUSSION THREAD

**Discussion by ykzts** [inabagumi/inaba-jinja#1436] [Ideas]
#198 と #235 の統合

くじ引きボタンを押した際に別のページに移動させるのではなくモーダル表示をさせるようにしたい。

ただしくじの進行状況に応じて `router.push` を使ってURLの変更は行うようにすることによってURLをシェアしてもらった際のアクセスがトップに集約されないようにする。

**Comment by haohanyang** [comment]
I recommend showing the process of the lottery without extra route and notifying the results in modal component

**Comment by ykzts** [comment]
Yes, I'm working on it.
